vassal-app/src/main/java/VASSAL/build/widget/HtmlChart.java

```diff
@@ -316,6 +327,7 @@ public View create(javax.swing.text.Element element) {
         if (kind != null && element.getName().equals("img")) {  //NON-NLS
           final String imageName = (String) element.getAttributes().getAttribute(HTML.Attribute.SRC);
           if (!imageName.contains("/")) {
+//            imageNames.add(imageName);
```

Comment:
?

Reply:
I'm not sure. I know I wasn't the one who put that comment in, because I don't comment things out "at column zero". I will take it out and hopefully Everything Will Be Great.

Reply:
Uh.... well actually "imagesNames", whatever it once was, doesn't exist. So Imma just put that comment back. Or we can delete the whole line entirely. NO IDEA.

Reply:
Oh yeah, I think `addLocalImageNames()` -- just above this -- handles all the actual adding of actual image tags to a list.

Reply:
That's probably a leftover from the code I started you with. My bad.